benchmark/dss/20200812/src/main/java/Application.java

```diff
@@ -0,0 +1,12 @@
+import io.github.ztkmkoo.dss.server.rest.DssRestServer;
```

Comment:
You'd better declare how to import this. (I think i cannot build this files)
It is better to add a maven or gralde config file.

Reply:
ah, I forgot this. Thank you.